tests/test_history.py

```diff
@@ -1386,7 +1453,7 @@ def make_equity_daily_bar_data(cls):
     @classmethod
     def create_df_for_asset(cls, start_day, end_day, interval=1,
                             force_zeroes=False):
-        sessions = cls.trading_calendar.sessions_in_range(start_day, end_day)
+        sessions = cls.nyse_calendar.sessions_in_range(start_day, end_day)
```

Comment:
You may want to consider adding the `WithTradingCalendars` fixture to this test, it provides `cls.trading_calendars[Equity]`.